build/tasks/aria-supported.js

```diff
@@ -0,0 +1,60 @@
+/*eslint-env node */
+'use strict';
+
+const { roles, aria: props } = require('aria-query');
+
+module.exports = function (grunt) {
+	grunt.registerMultiTask(
+		'aria-supported',
+		'Task for generating a diff of supported aria roles and properties.',
+		function () {
+			const entry = this.data.entry;
+			const destFile = this.data.destFile;
+			const listType = this.data.listType.toLowerCase();
+
+			/**
+			 * `axe` has to be dynamically required at this stage, as `axe` does not exist until grunt task `build:uglify` is complete, and hence cannot be required at the top of the file.
+			 */
+			const axe = require('../../axe');
+
+			const ariaKeys = Array.from(props).map(([key]) => key)
```

Comment:
Where is `props` coming from here?

Reply:
This line (no. 4) - const { roles, aria: props } = require('aria-query');
es6 destructuring